core/src/test/java/com/google/common/truth/PrimitiveByteArraySubjectTest.java

```diff
@@ -85,7 +84,8 @@ public void isEqualTo_Fail_UnequalOrdering() {
       assertThat(e)
           .hasMessageThat()
           .isEqualTo(
-              "Not true that <(byte[]) [0, 1]> is equal to <[1, 0]>; "
+              "Not true that <[0,1> is equal to <[1,0>; "
```

Comment:
If possible, I think the same message it was printing before should be preserved here (maybe with the exception of the `(byte[])`; it's not clear to me yet if it should be kept or not).